packages/mcp9808/mcp9808.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
+// PCF8591 driver for CHIRIMEN
+// based from https://github.com/adafruit/Adafruit_MCP9808_Library/blob/master/Adafruit_MCP9808.cpp
+// Programmed by Haruka Terai
+
+const MCP9808_I2CADDR_DEFAULT= 0x18;      ///< I2C address
+const MCP9808_REG_CONFIG= 0x01;           ///< MCP9808 config register
+const MCP9808_REG_CONFIG_SHUTDOWN= 0x0100 ///< shutdown config
+const MCP9808_REG_AMBIENT_TEMP= 0x05      ///< ambient temperature
+const MCP9808_REG_RESOLUTION= 0x08        ///< resolution
+
+class MCP9808{
+    constructor(i2cPort,slaveAddress){
+    this.i2cPort = i2cPort;
+    this.i2cSlave = null;
+    this.slaveAddress = slaveAddress;
+  }
+  async init() {
+    this.i2cSlave = await this.i2cPort.open(this.slaveAddress);
+    await this.write16(MCP9808_REG_CONFIG,0x0);
+  }
+  async readTempC(){
+    let temp = null;
+    let t = await this.read16(MCP9808_REG_AMBIENT_TEMP);
```

Comment:
t は、初回の代入しかしていなのであれば、const で良いと思いました。
他の関数 / メソッドも同様
```suggestion
    const t = await this.read16(MCP9808_REG_AMBIENT_TEMP);
```